### DISCUSSION TITLE
Noise as output

### DISCUSSION THREAD

**Discussion by bvhari** [ljleb/comfy-mecha#17] [Q&A] [answered]
Whenever I try to use these nodes, I get noise as the output. Even a simple Recipe -> Merger -> KSampler gives noise. I got a valid image only once randomly. Any tips on fixing this issue?
I'm on Comfy commit 6c9dbde

**Comment by ljleb** [comment]
This could be happening for different reasons. Could you please share the workflow you used? It would help reproducing the issue. Does it generate noise even when the recipe is literally a single model?

**Reply by bvhari** [reply, marked as answer]
I think I figured out the issue
I had changed the device and dtype settings, looks like it _has_ to be fp64 for the merge dtype
It's working now 👍

**Reply by ljleb** [reply]
AFAIK, this depends on the specific recipe you're using. For example, a simple weighted sum should be completely fine in FP16. If that's not the case on your side, there might be a problem somewhere.
